eras/conway/impl/src/Cardano/Ledger/Conway/TxInfo.hs

```diff
@@ -764,3 +769,30 @@ class
 
 instance ConwayEraPlutusTxInfo 'PlutusV3 ConwayEra where
   toPlutusChangedParameters _ x = PV3.ChangedParameters (PV3.dataToBuiltinData (toPlutusData x))
+
+-- | Translate a validity interval to POSIX time
+transValidityInterval ::
+  forall proxy era a.
+  Inject (AlonzoContextError era) a =>
+  proxy era ->
+  EpochInfo (Either Text) ->
+  SystemStart ->
+  ValidityInterval ->
+  Either a PV1.POSIXTimeRange
+transValidityInterval _ epochInfo systemStart = \case
```

Comment:
```suggestion
conwayTransValidityInterval ::
  forall proxy era a.
  Inject (AlonzoContextError era) a =>
  proxy era ->
  EpochInfo (Either Text) ->
  SystemStart ->
  ValidityInterval ->
  Either a PV1.POSIXTimeRange
conwayTransValidityInterval _ epochInfo systemStart = \case
```

Reply:
Same here. It would be inconsistent how we deal with TxInfo